Source/ToolCore/Build/ResourcePackager.cpp

```diff
@@ -201,11 +201,6 @@ void ResourcePackager::GeneratePackage(const String& destFilePath)
             return;
         }
 
-        if (!file.GetSize())
-        {
-            return;
-        }
-
```

Comment:
Yup, having it exclude 0b items was pretty simple too (just remove them from the list at this point). Is there any reason you'd prefer to include them?

The build system is going to need something a bit more complex than just including all files at some point. It's quite reasonable to expect there to be files in the resources directory that are for testing only or need to be excluded for some other reason. Unity's solution is to explicitly specify scenes in your build settings and it determines what to include from there, I think Unreal does something similar. A whitelist or blacklist (probably simpler) would probably be just fine though, and would make for customizable builds by passing different lists to the build process. The "default" list could be modified by marking particular resources (or even directories) as excluded in the inspector for those objects. .